dvc/utils/serialize/_toml.py

```diff
@@ -1,9 +1,46 @@
+import logging
 from contextlib import contextmanager
 
 from funcy import reraise
 
 from ._common import ParseError, _dump_data, _load_data, _modify_data
 
+logger = logging.getLogger(__name__)
+
+
+def _tomlkit_to_popo(d):
+    """Converts tomlkit objects to plain old python objects (popo)
+    We could probably just do dict(tmlkit_item.items()) but this
+    seems more robust since it handles lists
+
+    Taken from https://github.com/sdispater/tomlkit/issues/43
+    """
```

Comment:
Feels like this is really something that belongs in an upstream PR for tomlkit and not DVC

Reply:
I have this part done now. I am working on another aspect now